TravelerKitUI/ViewControllers/BookableDetailsViewController.swift

```diff
@@ -190,6 +190,8 @@ extension BookableDetailsViewController: AvailabilityCheckDelegate {
     func availabilityCheckDidFailWith(_ error: Error) {
         tableView.isUserInteractionEnabled = true
 
+        errorContext?.error = BookingError.noDate
```

Comment:
This is not the same error.. noDate means date has not been provided.

Error context is not necessary for this callback.

Is there a reason you added this?

Reply:
I can add a new error to describe no network connectivity

When the availability check fails, the errorContext is not updated and the screen is not reupdated. 

https://www.useloom.com/share/a1434a21f9b94d198e3f1536cd8ae293

At the end of the video there is no network connectivity, when the user tries to close the date picker by clicking on the date row, the app crashes because the amount of rows available is not updated.